### PR TITLE
NASA Image and Video Library (Independent Publisher)

### PR DESCRIPTION
**Pull request by paulculmsee**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

- [x ] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [x] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [x ] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 

image

image

image

- [x] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.

image
image

- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

File: independent-publisher-connectors/NASA Image and Video Library/apiDefinition.swagger.json [hunk 1]
```diff
@@ -0,0 +1,597 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "NASA Image and Video Library",
+    "description": "Use this API to access the NASA Image and Video Library site at images.nasa.gov",
+    "version": "1.0",
+	"contact": {
```
Comment (sriyen-msft) [inline independent-publisher-connectors/NASA Image and Video Library/apiDefinition.swagger.json:7]: @paulculmsee Indentation - The formatting of both the swagger and apiProperties.json files is not correct. The properties are not aligned properly and there is a longer tab index between the properties. Please refer to the certified connectors in the [open source repo](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/certified-connectors/Africa%27s%20Talking%20Airtime/apiDefinition.swagger.json) for reference.

File: independent-publisher-connectors/NASA Image and Video Library/apiDefinition.swagger.json [hunk 2]
```diff
@@ -0,0 +1,597 @@
...
+                                  },
+                                  "description": "Media keywords"
+                                },
+                                "media_type": {
+                                  "type": "string",
+                                  "description": "Media Type"
+                                },
+                                "nasa_id": {
+                                  "type": "string",
+                                  "description": "The media asset\u2019s NASA ID"
```
Comment (sriyen-msft) [inline independent-publisher-connectors/NASA Image and Video Library/apiDefinition.swagger.json:79]: @paulculmsee Unicodes - The connector's swagger still have unicodes which may not be supported in all the Power Automate page, Can you please replace all the unicodes with respective characters in the English language.

File: independent-publisher-connectors/NASA Image and Video Library/apiDefinition.swagger.json [hunk 3]
```diff
@@ -0,0 +1,597 @@
...
+            "type": "string",
+            "description": "NASA center which published the media.",
+            "x-ms-summary": "NASA center"
+          },
+          {
+            "name": "description",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "description": "Terms to search for in \u201cDescription\u201d fields.",
```
Comment (sriyen-msft) [inline independent-publisher-connectors/NASA Image and Video Library/apiDefinition.swagger.json:187]: @paulculmsee Unicodes - The connector's swagger still have unicodes which may not be supported in all the Power Automate page, Can you please replace all the unicodes with respective characters in the English language.

Applies the same comment to all the unicodes in the connector artifacts.

File: independent-publisher-connectors/NASA Image and Video Library/apiProperties.json [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+          "tooltip": "Provide your API Key",
+          "constraints": {
+            "tabIndex": 2,
+            "clearText": false,
+            "required": "true"
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#0B3D91",
```
Comment (sriyen-msft) [inline independent-publisher-connectors/NASA Image and Video Library/apiProperties.json:18]: @paulculmsee The iconBrandColor for all the independent publisher connectors should be `"#da3b01"`.

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

File: independent-publisher-connectors/NASA Image and Video Library/apiDefinition.swagger.json (hunk 3 above)
Comment (paulculmsee) [reply independent-publisher-connectors/NASA Image and Video Library/apiDefinition.swagger.json]: done

File: independent-publisher-connectors/NASA Image and Video Library/apiProperties.json (hunk 4 above)
Comment (paulculmsee) [reply independent-publisher-connectors/NASA Image and Video Library/apiProperties.json]: done

**Comment by sriyen-msft** [comment]
[[certify-connector]]

**Review by sriyen-msft — APPROVED** [review]

**Pull request merged**